react-app/src/components/TopNavigationBar/TopNavigation.css

```diff
@@ -0,0 +1,132 @@
+.profile-dropdown {
+    position: absolute;
+   
```

Comment:
coding style comment - its good practice to keep spacing consistent within code blocks and between code blcoks